harness/determined/searcher/_search_runner.py

```diff
@@ -194,18 +194,25 @@ def post_operations(
         )
         # TODO: Remove this temporary hack in favor of an actual solution around the issue described
         # in https://determined-ai.slack.com/archives/C04645NHSP6/p1680201334658789
+        # Also verify that this is actually catching the error.
+        # Remove print tests.
         try:
             bindings.post_PostSearcherOperations(
                 session,
                 body=body,
                 experimentId=experiment_id,
             )
         except errors.APIException as e:
+            print(f"Catching errors.APIException: {str(e)}")
```

Comment:
don't these need to be `logging.info`? didn't know prints work. If so, no problem. I want to look into this issue at some point too because the fact we are even doing this try-except seems very wrong to me

Reply:
they work, but I changed them to logging.warnings to make them stand out more. obviously to be removed, eventually.

definitely very wrong, just a band aid